Mlem/MlemApp.swift

```diff
@@ -21,6 +22,15 @@ struct MlemApp: App {
             Window()
                 .environmentObject(accountsTracker)
                 .onAppear {
+                    var imageConfig = ImagePipeline.Configuration.withDataCache(name: "main", sizeLimit: AppConstants.cacheSize)
```

Comment:
looking at the responses we get from images from Lemmy they contain `cache-control` values, and according to the documentation for this library the default caching mechanism supports that, here we're changing it to use the type of caching that ignores those headers, do we need to?

Reply:
I didn’t know lemmy returns cache-control, so I choose a cache type that ignores it, 

it makes me feel safer if we’d control those images and not those servers (considering how the api is built) but if you’d like I can change it